### PR TITLE
dx12: Block compressed format support for buffer-image copies

### PR DESCRIPTION
**Pull request by msiglreith**
Tested against a few of the vulkan samples, hit every path at least once iirc.

### CODE REVIEW

File: src/backend/dx12/src/command.rs [hunk 1]
```diff
@@ -38,8 +38,11 @@ fn get_rect(rect: &pso::Rect) -> d3d12::D3D12_RECT {
 }
 
 fn div(a: u32, b: u32) -> u32 {
-    assert_eq!(a % b, 0);
-    a / b
+    (a + b - 1) / b
```
Comment (kvark) [inline src/backend/dx12/src/command.rs]: oh wow, this would change the semantics completely, wouldn't it?

Comment (grovesNL) [reply src/backend/dx12/src/command.rs]: It's just `ceil` of `a / b`

Comment (kvark) [reply src/backend/dx12/src/command.rs]: Heh, I misread. Indeed fine is it. /yoda

**Pull request merged**